plugins/src/messages.rs

```diff
@@ -63,6 +63,30 @@ pub struct GetManifestCall {}
 #[derive(Deserialize, Debug)]
 pub(crate) struct InitCall {
     pub(crate) options: HashMap<String, Value>,
+    pub(crate) configuration: Configuration,
+}
+
+#[derive(Clone, Deserialize, Debug)]
+pub struct Configuration {
+    #[serde(rename = "lightning-dir")]
+    pub lightning_dir: String,
+    #[serde(rename = "rpc-file")]
+    pub rpc_file: String,
+    pub startup: bool,
+    pub network: String,
+    pub feature_set: HashMap<String, String>,
+    pub proxy: ProxyInfo,
+    #[serde(rename = "torv3-enabled")]
+    pub torv3_enabled: bool,
+    pub always_use_proxy: bool,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct ProxyInfo {
+    #[serde(alias = "type")]
+    pub tup: String,
```

Comment:
```suggestion
    pub typ: String,
```

This is my typo, sorry!

Reply:
Done